flow/order/refund.yaml

```
@@ -1,2 +1,61 @@
 steps:
-  - say: refund
+  - if: (@ flow.get('selected_order') )
+    then:
+      jump: selected order
+    else: next
+
+  - type: meya.shopify.component.customer.order.list
+    status: any
+    customer_id: (@ user.user_id )
+    integration: integration.shopify
+  - flow_set: refund_orders
+
+  - say: Cheking orders....
```

Move to where the operation actually happens

```suggestion
  - say: Checking orders....
```